crates/qt-build-utils/src/lib.rs

```diff
@@ -306,8 +306,82 @@ impl QtBuild {
         .to_string()
     }
 
+    fn cargo_link_qt_library(
+        &self,
+        builder: &mut cc::Build,
+        name: &str,
+        prefix_path: &str,
+        lib_path: &str,
+        link_lib: &str,
+        prl_path: &str,
+    ) {
+        println!("cargo:rustc-link-lib={link_lib}");
+
+        match std::fs::read_to_string(prl_path) {
+            Ok(prl) => {
+                for line in prl.lines() {
+                    if let Some(line) = line.strip_prefix("QMAKE_PRL_LIBS = ") {
+                        parse_cflags::parse_libs_cflags(
+                            builder,
+                            name,
+                            line.replace(r"$$[QT_INSTALL_LIBS]", lib_path)
+                                .replace(r"$$[QT_INSTALL_PREFIX]", prefix_path)
+                                .as_bytes(),
+                        );
+                    }
+                }
+            }
+            Err(e) => {
+                println!(
+                    "cargo:warning=Could not open {} file to read libraries to link: {}",
+                    &prl_path, e
+                );
+            }
+        }
+    }
+
+    /// Some prl files don't follow a consistent naming scheme. Try to find it by looking at all files in lib_path.
```

Comment:
Can you be more specific in this comment? What is an example of a filename that doesn't match the usual pattern?

Reply:
Should be done please review